src/v/kafka/server/handlers/configs/config_utils.h

```diff
@@ -69,6 +70,61 @@ T make_error_alter_config_resource_response(
       .resource_type = resource.resource_type,
       .resource_name = resource.resource_name};
 }
+/**
+ * Authorizes groupped alter configuration resources, it returns not authorized
+ * responsens and modifies passed in group_resources<T>
+ */
+template<typename T, typename R>
+std::vector<R> authorize_alter_config_resources(
```

Comment:
do we want a quick test for this partitioning function ? how are we thinking of testing this.

Reply:
I think we will create a whole suite of tests for authorization

Reply:
sounds good, please follow up w/ @dotnwat for the ducktape tests for this.